test/irb/test_init.rb

```diff
@@ -60,6 +60,22 @@ def test_rc_file_in_subdir
       ENV["IRBRC"] = backup_irbrc
     end
 
+    def test_no_color_environment_variable
+      orig = ENV['NO_COLOR']
+
+      assert IRB.conf[:USE_COLORIZE]
+
+      ENV['NO_COLOR'] = 'true'
+      IRB.setup(eval("__FILE__"))
+      refute IRB.conf[:USE_COLORIZE]
+
+      ENV['NO_COLOR'] = nil
+      IRB.setup(eval("__FILE__"))
+      assert IRB.conf[:USE_COLORIZE]
```

Comment:
Why `eval` for `__FILE__` in the same context?

```suggestion
      IRB.setup(__FILE__)
      refute IRB.conf[:USE_COLORIZE]

      ENV['NO_COLOR'] = nil
      IRB.setup(__FILE__)
      assert IRB.conf[:USE_COLORIZE]
```